src/utilities/api.ts

```diff
@@ -1,4 +1,4 @@
-import axios from "axios";
+import Axios from "axios";
```

Comment:
I saw that "axios" and "Axios" are different classes inside the axios module. What is the difference between those two?

Reply:
The name doesn't matter in the file which imports the default export,... so inside this file it's the same just aliased.

Reply:
So you just like the big letter A better than lower a 🤔

Reply:
yes because the types and functions we need is actually from the internal upper A Axios